frontend/src/features/mapSideBar/SidebarContents/AssociatedLandForm.tsx

```diff
@@ -291,7 +291,7 @@ interface IAssociatedLandForm {
   /** The initial building values to add associated land to */
   initialValues: IBuilding;
   /** Whether or not this user has admin privileges */
```

Comment:
in the Landform comment you changed "admin" to "SRES" here, you might want to do that here too?